src/pages/components/menu.mdx

```diff
@@ -132,11 +131,57 @@ import MenuMediaCode from '!!raw-loader!../../examples/dropdowns/MenuMedia.tsx';
 
 ## Configuration
 
-<Configuration reactPackage={props.data.mdx.package} propSheets={props.data.mdx.propsSheets} />
+<Configuration reactPackage={props.data.mdx.package} components={props.data.mdx.components} />
 
 ## API
 
-<PropSheets propSheets={props.data.mdx.propsSheets} />
+### Menu
+
+<PropSheet components={props.data.mdx.components} componentName="menu" />
+
+### Separator
+
+<PropSheet components={props.data.mdx.components} componentName="separator" />
+
+### AddItem
+
+<PropSheet components={props.data.mdx.components} componentName="addItem" />
+
+### HeaderIcon
+
+<PropSheet components={props.data.mdx.components} componentName="headerIcon" />
+
+### HeaderItem
+
+<PropSheet components={props.data.mdx.components} componentName="headerItem" />
+
+### Item
+
+<PropSheet components={props.data.mdx.components} componentName="item" />
+
+### ItemMeta
+
+<PropSheet components={props.data.mdx.components} componentName="itemMeta" />
```

Comment:
Should component names be PascalCased?

Reply:
It doesn't matter – they can be any-cased. I just thought pascalCased looked best.